bundles/binding/org.openhab.binding.comfoair/README.md

```diff
@@ -1,458 +1,1020 @@
-        }
-end
-
-rule "Manueller Eingriff"
-when
-    Item Lueftung_Fan_Level changed
-then
-
-    if( autoChangeInProgress ){
-
-        autoChangeInProgress = false
-    }
-    else{
-
-        postUpdate(Lueftung_Auto_Mode,0)
-    }
-end
-
-rule "Lüfterstufe"
-when
-    Item Lueftung_Auto_Mode changed
-    or
-    Time cron "0 0/1 * * * ?"
-then
-    if( (Lueftung_Auto_Mode.state as DecimalType) == 1 ){
-
-        var Number day    = now.getDayOfWeek
-        var Number hour   = now.getHourOfDay
-        var Number minute = now.getMinuteOfHour
-
-        var boolean isNight = false
-
-        // Freitag
-        if( day == 5 ){
-            if( hour < 7 || (hour >= 23 && minute >=30) ) isNight = true
-        }
-        // Samstag
-        else if( day == 6 ){
-            if( hour < 9 || (hour >= 23 && minute >=30) ) isNight = true
-        }
-        // Sonntag
-        else if( day == 7 ){
-            if( hour < 9 || (hour >= 22 && minute >=30) || hour >= 23 ) isNight = true
-        }
-        else{
-
-            if( hour < 7 || (hour >= 22 && minute >=30) || hour >= 23 ) isNight = true
-        }
-
-        var Number currentLevel = (Lueftung_Fan_Level.state as DecimalType)
-        var Number newLevel = 3
-
-        if( !isNight ){
-
-            var Number raumTemperatur = (Lueftung_Ablufttemperatur.state as DecimalType)
-            var Number aussenTemperatur = (Lueftung_Aussenlufttemperatur.state as DecimalType)
-            var Number zielTemperatur = (Lueftung_Komfortemperatur.state as DecimalType)
-
-            if(
-                raumTemperatur >= zielTemperatur
-                &&
-                aussenTemperatur >= raumTemperatur
-            ){
-
-                newLevel = 1
-            }
-            else if(
-                raumTemperatur >= zielTemperatur - 1
-                &&
-                aussenTemperatur >= raumTemperatur - 1
-                &&
-                currentLevel == 1
-            ){
-
-                newLevel = 1
-            }
-        }
-        else if( Fenster_OG_Schlafzimmer.state == CLOSED || Fenster_OG_Ankleide.state == OPEN ) {
-
-            newLevel = 2
-        }
-
-        if( newLevel != currentLevel ){
-
-            if( newLevel == 1 ){
-
-                logInfo("airflow.rules", "auto slowdown start")
-                    send("test@gmail.com", "Lüftung verlangsamt")
-            }
-            else if( currentLevel == 1 ){
-
-                    logInfo("airflow.rules", "auto slowdown end")
-                send("test@gmail.com", "Lüftung wieder normal")
-            }
-
-            autoChangeInProgress=true
-            sendCommand(Lueftung_Fan_Level,newLevel)
-        }
-    }
-end
-```
+# ComfoAir Binding modified
+Expanded ComfoAir binding v.1x for openHAB based on original [org.openhab.binding.comfoair](https://github.com/openhab/openhab1-addons/wiki/comfo-air-binding) v. 1.3.0 written by Holger Hees.

```

Comment:
Please replace this sentence with a brief summary of what the binding is and/or does.

Reply:
Done.